fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java

```diff
@@ -953,6 +969,16 @@ public void afterCommitted(TransactionState txnState, boolean txnOperated) throw
         if (!txnOperated) {
             return;
         }
+
+        // sync stream load collect profile
+        if (isSyncStreamLoad() && Config.enable_stream_load_profile) {
```

Comment:
The stream load may have thousands of profiles. It may consume much memory in the FE.
You need to know how much size one profile is and how many profiles you want to store.